tExtRef/doesFcdaMeetExtRefRestrictions.ts

```diff
@@ -26,10 +33,13 @@ export function doesFcdaMeetExtRefRestrictions(
 
   if (
     extRef.getAttribute("pServT") &&
-    controlBlockType !== extRef.getAttribute("pServT")
+    options?.controlBlockType !== extRef.getAttribute("pServT")
```

Comment:
```suggestion
    options.controlBlockType !== extRef.getAttribute("pServT")
```

I guess we don't need the question mark.

Reply:
It seems a bit odd because the `controlBlockType` is not really optional.
If `pServT` is declared on the `extRef` we carry out this check and if `options.controlBlockType` is not declared I think we are checking "undefined" against the `pServT` value.

Does that sound right?

If it was really optional to check the control block type then that makes the `pServT` check also optional. In which case should the option be for the `pServT` check? Or should we be defaulting to not check the `pServT` if a control block isn't given (my preference).

In which case something like:

```js
if (
    extRef.getAttribute("pServT") && controlBlockType &&
    controlBlockType !== extRef.getAttribute("pServT")
    options?.controlBlockType !== extRef.getAttribute("pServT")
  )
```

I realise this hasn't functionally changed as part of this PR but I think it is worth discussing.

Reply:
So far it was very restrictive with missing control tag input. I think there are two ways to go:
1. Make `controlBlockType` not optional and force through typescript
2. Interpret missing `controlBlockType` definition as user does not want to check for `pServT`.

I have implemented the second now. Would love to heard which one you would prefer.

Reply:
I am good with Option 2.
Thank you for considering this feedback :+1: